src/seedu/addressbook/Main.java

```diff
@@ -55,7 +56,7 @@ private void start(String[] launchArgs) {
             this.addressBook = storage.load();
             ui.showWelcomeMessage(VERSION, storage.getPath());
 
-        } catch (InvalidStorageFilePathException | StorageOperationException e) {
+        } catch (InvalidStorageFilePathException | StorageOperationException | IOException e) {
```

Comment:
Consider creating the file and showing the problem to the user, instead of closing the program.